src/machine/machine_atsamd51.go

```diff
@@ -1734,17 +1734,19 @@ func (pwm PWM) getMux() PinMode {
 
 // USBCDC is the USB CDC aka serial over USB interface on the SAMD21.
 type USBCDC struct {
-	Buffer  *RingBuffer
-	TxIdx   volatile.Register8
-	waitTxc bool
-	sent    bool
+	Buffer       *RingBuffer
+	TxIdx        volatile.Register8
+	waitTxc      bool
+	waitTxcTimer uint8
```

Comment:
Perhaps this might be better named `waitTxcRetryCount` ?

Reply:
Thank you.
You're right, it's retry, not timeout.